cg/constants/gene_panel.py

```diff
@@ -64,6 +64,18 @@ def collaborators() -> set[str]:
             CustomerId.CUST042,
         }
 
+    @staticmethod
+    def is_customer_collaborator_and_panels_in_gene_panels_master_list(
+        customer_id: str, gene_panels: set[str]
+    ) -> bool:
+        return customer_id in GenePanelMasterList.collaborators() and gene_panels.issubset(
+            GenePanelMasterList.get_panel_names()
+        )
+
+    @staticmethod
+    def get_broad_non_specific_gene_panels() -> set[str]:
```

Comment:
```suggestion
    def get_non_specific_gene_panels() -> set[str]:
```
I think broad and non-specific are a tautology here.

Reply:
Fixed